src/app/checklists/checklist-tree/checklist-tree.component.ts

```diff
@@ -144,11 +146,20 @@ export class ChecklistTreeComponent implements OnInit, AfterViewInit {
     } else {
       if (node.group) {
         // Adding new checklist to a group.
+        checklistGroup = node.group;
+
         checklist = Checklist.create();
-        if (!(await this._fillTitle(checklist, 'checklist'))) {
+
+        // Reproduce Garmin Pilot's default for actions.
```

Comment:
I think I have finally understood how it all plays together after re-reading the complete code in one go (so please disregard the original comments, but I'm not deleting them to avoid further confusion).

I believe this really **does** work due to the default being the first item in the proto definition (`ACTION_DO_NOTHING` and not `ACTION_UNKNOWN`, which doesn't exist in this case).

Reply:
Yup, that's how proto works (and why some people are picky about what the 0 value of an enum is named). 

What this means is that if you have an existing file in your browser's storage, or when you import a file with another format, it'll load with all DO_NOTHING actions, and that'll be explicitly saved back when it's saved again. If I defined an UNKNOWN value then I'd need to explicitly handle it in multiple places.